### ISSUE TITLE
Upload PyInstaller Bootloaders to VirusTotal Automatically

### ISSUE THREAD

**Issue by danyeaw** [pyinstaller/pyinstaller#8372]
**Is your feature request related to a problem? Please describe.**
Virus tool vendors are still continuing to create false positives against PyInstaller's bootloader. This is a shame because the antivirus tool vendors should really be better at detecting viruses, however, I had an idea on how we could at least reduce the false positives.

**Describe the solution you'd like**
We started using VirusTotal to upload built versions of Gaphor through CI/CD. Although this hasn't totally made the problem go away, I have noticed that VirusTotal is sharing these results with the antivirus vendors, and it seems to be reducing the number of false positives over time. In fact, I just built the PyInstaller bootloaders on Windows with clang, gcc, and msvc with various other settings like debug on and off. Only the gcc ones are showing clean currently - which matches the version we have been uploading.

Either daily, or every time we do a build on the develop branch, build all the bootloaders for each platform and with different compilers and options. Upload all the bootloaders to VirusTotal using their API key. There is a social aspect to this as well which is optional, where we could add comments to the files that they are good, and the files could get positive reputation over time.

**Describe alternatives you've considered**
Tradeoffs I have thought about is that VirusTotal allows for free API access, but they are part of Google. They also keep the files uploaded and share the files with antivirus companies, which has some privacy implications. However, this sharing would have a net benefit for us, since it would likely reduce false positives.

We could also keep the status quo and not do this.

**Additional context**
I'm happy to contribute / set up these changes if there is agreement that this is something PyInstaller wants to get involved in.

**Comment by bwoodsend** [comment]
We did do this [once upon a time](https://github.com/bwoodsend/pyinstaller-malware-analysis/). I find it hard to believe that just submitting files for analysis would be enough to make vendors reevaluate their assessments but if all I have to do to test that theory is to re-enable a workflow then we might as well do it.

The comments idea is interesting though (sources since I had to look them up: [concept](https://docs.virustotal.com/reference/comments), [API](https://docs.virustotal.com/reference/files-comments-post)).
